src/index.html

```diff
@@ -29,7 +29,7 @@
 
   <title>HiGlass Pileup Track</title>
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css">
-  <link rel="stylesheet" href="hglib.css">
+  <link rel="stylesheet" href="https://unpkg.com/higlass@1.12.2/dist/hglib.css">
```

Comment:
This could potentially be problematic if one wanted to use a local development version of higlass to do development. But I concede that that's an edge case.